src/Symfony/Component/HttpFoundation/Session/Storage/MockArraySessionStorage.php

```diff
@@ -75,8 +73,6 @@ public function __construct($name = 'MOCKSESSID', MetadataBag $metaBag = null)
     }
 
     /**
-     * Sets the session data.
```

Comment:
I would only keep the `@var array|SessionBagInterface[]` change, which adds value.